### ISSUE TITLE
Will you rename YOLOv5 to Fast-YOLOv4?

### ISSUE THREAD

**Issue by Lornatang** [ultralytics/yolov5#87]
The work you have done is great, but in the face of doubts from the community, I can only express my sympathy.

**Comment by Louis24** [comment]
OK

**Comment by rcg12387** [comment]
Fast-YOLOv4? Is this faster than [YOLOv4](https://github.com/AlexeyAB/darknet/issues/6067)?

**Comment by Louis24** [comment]
> Fast-YOLOv4? Is this faster than [YOLOv4](https://github.com/AlexeyAB/darknet/issues/6067)?

yeah

**Comment by rcg12387** [comment]
@Louis24 Then what about [this?](https://github.com/ultralytics/yolov5/issues/6#issuecomment-643823425)

**Comment by rcg12387** [comment]
@Louis24 Not at all. No faster, no better than YOLOv4. Besides, [YOLOv4-tiny](https://github.com/AlexeyAB/darknet/issues/6067) is the fastest yet has better performance than YOLOv5s.
In my opinion, at now the unique merit of YOLOv5 is that it is implemented in pytorch. Anyway, it is the on-going repo and very good news for the pytorch community including me.

**Comment by glenn-jocher** [comment]
@rcg12387 yolov4-tiny performs at 40.2 AP50 according to your link, while yolov5s (this repo) performs at 55.8 AP50. You can see this in our readme table: https://github.com/ultralytics/yolov5#pretrained-checkpoints

**Comment by rcg12387** [comment]
@glenn-jocher Ouch, I was mislead. I thought it was AP50 in your benchmark graph. Very sorry.

**Comment by glenn-jocher** [comment]
@rcg12387 thanks, its ok! YOLO (i.e. pj reddie) typically emphasized AP50 more than the AP50:95 that the community more often uses these days. We've tried to align our training for best AP50:95 mAP to more properly measure against the other detectors out there like EfficientDet, FCOS etc. that primarily use this metric.

**Comment by Kreijstal** [comment]
"YOLOv5" should be renamed YOLOtorch or something like that
